### PR TITLE
Clean: Remove SVN references in build (OFBIZ-12267)

### PR DESCRIPTION
**Pull request by ieugen**
Improved: Remove SVN references in build (OFBIZ-12267)

Removed SVN references in git ant templates.

### CODE REVIEW

**Review by JacquesLeRoux — COMMENTED** [review]
SvnCheckout and svntools should not be removed. They are still used by pullPluginSource and pullAllPluginsSource tasks

File: build.gradle [hunk 1]
```diff
@@ -16,7 +16,6 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import at.bxm.gradleplugins.svntools.tasks.SvnCheckout
 import org.apache.tools.ant.filters.ReplaceTokens
 import org.asciidoctor.gradle.AsciidoctorTask
```
Comment (JacquesLeRoux) [inline build.gradle]: SvnCheckout and svntools should not be removed. They are still used by pullPluginSource and pullAllPluginsSource tasks which should also not be removed.

File: build.gradle [hunk 2]
```diff
@@ -959,11 +922,6 @@ task pullAllPluginsSource(group: ofbizPlugin,
     task deleteBeforePulling {
         doLast { delete "${pluginsDir}" }
     }
-    task pullPluginsFromSvn(type: SvnCheckout, dependsOn: deleteBeforePulling) {
```
Comment (JacquesLeRoux) [inline build.gradle:962]: pullPluginsFromSvn should also not be removed. It uses a GH special feature.

**Comment by JacquesLeRoux** [comment]
Hi Eugen,

Thanks for the work, small changes pending, TIA

**Comment by PierreSmits** [comment]
In order to stop confusion and unnecessary workload renaming should be considered.

**Comment by ieugen** [comment]
@JacquesLeRoux I updated the commit.
@PierreSmits : added mentions to GH SVN . 
Did not know about that.

File: themes/rainbowstone/README.md [hunk 3]
```diff
@@ -4,10 +4,10 @@
 # Rainbowstone theme
 This back-end OFBiz theme is bootstrap based and offers 4 colour variants. As an example
 
-<img src="http://svn.apache.org/repos/asf/ofbiz/ofbiz-framework/trunk/themes/rainbowstone/webapp/rainbowstone/images/themeRuby.png" alt="Rainbowstone ruby colour variant" />
+<img src="https://raw.githubusercontent.com/apache/ofbiz-framework/trunk/themes/rainbowstone/webapp/rainbowstone/images/themeRuby.png" alt="Rainbowstone ruby colour variant" />
 
 ## License
-The Rainbowstone theme uses the Quicksand font which is licensed under the [SIL OPEN FONT LICENSE Version 1.1](https://svn.apache.org/repos/asf/ofbiz/ofbiz-framework/trunk/themes/rainbowstone/webapp/rainbowstone/fonts/quicksand/Quicksand SIL OFL License 1.1.txt "SIL OPEN FONT LICENSE Version 1.1")
+The Rainbowstone theme uses the Quicksand font which is licensed under the [SIL OPEN FONT LICENSE Version 1.1](https://raw.githubusercontent.com/apache/ofbiz-framework/trunk/LICENSE)
```
Comment (dram) [inline themes/rainbowstone/README.md:10]: This line may need to be updated, as mention of SIL OFL in LICENSE is removed recently in commit 180b24.

Comment (JacquesLeRoux) [reply themes/rainbowstone/README.md:10]: Thanks Wang,

I made some changes at 
https://github.com/apache/ofbiz-framework/commit/a80c12ae3344c2c899564e5f75dfd975240c9115/ 
+
https://github.com/apache/ofbiz-framework/commit/2cf1e9d490234e7758c0d1b937d50eaa0367173d/

**Pull request merged**